src/cc65/expr.h

```diff
@@ -54,23 +54,23 @@ int evalexpr (unsigned flags, void (*Func) (ExprDesc*), ExprDesc* Expr);
 void Expression0 (ExprDesc* Expr);
 /* Evaluate an expression via hie0 and put the result into the primary register */
 
-void ConstExpr (void (*Func) (ExprDesc*), ExprDesc* Expr);
-/* Will evaluate an expression via the given function. If the result is not
-** a constant of some sort, a diagnostic will be printed, and the value is
-** replaced by a constant one to make sure there are no internal errors that
-** result from this input error.
-*/
-
 void BoolExpr (void (*Func) (ExprDesc*), ExprDesc* Expr);
 /* Will evaluate an expression via the given function. If the result is not
 ** something that may be evaluated in a boolean context, a diagnostic will be
 ** printed, and the value is replaced by a constant one to make sure there
 ** are no internal errors that result from this input error.
 */
 
-void ConstAbsIntExpr (void (*Func) (ExprDesc*), ExprDesc* Expr);
-/* Will evaluate an expression via the given function. If the result is not
-** a constant numeric integer value, a diagnostic will be printed, and the
+ExprDesc StaticConstExpr (void (*Func) (ExprDesc*));
+/* Get an expression evaluated via the given function. If the result is not a
+** static constant expression, a diagnostic will be printed, and the value is
```

Comment:
What is a "static constant expression"? Explain or include pointers to which definitions of these terms you're using.

Reply:
It could be read at the definition of `E_EVAL_C_CONST`. `Static` in such a context means "evaluated without any runtime code generated" as in #1196. What about changing the phrase
> static constant expression

to
> constant expression needing no runtime code

?